contracts/dao/nopeDAO/NopeDAO.sol

```diff
@@ -28,7 +28,7 @@ contract NopeDAO is
         Governor("NopeDAO")
         GovernorSettings(
             0, /* 0 blocks */
-            4 days,
+            26585, /* 4 days measured in blocks. Assumed 13s block time */
```

Comment:
Using blocks to measure time is an antipattern. I haven't looked at the GovernorSettings, but is there a way we can use block.timestamp instead of # of blocks?

Reply:
I agree it's an antipattern and is what caught me out originally. The `GovernorSettings` module measures time in units of blocks unfortunately